difference provides both a library and a binary, but the binary is only built
properly if the non-default bin feature is set.  As we cannot select different
features for the library and binary build, we make bin a default feature.
Index: difference/Cargo.toml
===================================================================
--- difference.orig/Cargo.toml
+++ difference/Cargo.toml
@@ -36,7 +36,7 @@ version = ">= 0.2.7, < 0.6"
 
 [features]
 bin = ["getopts"]
-default = []
+default = ["bin"]
 [badges.appveyor]
 repository = "johannhof/difference.rs"
 
